.github/workflows/build.yml

```diff
@@ -73,12 +93,14 @@ jobs:
         shell: bash
```

Comment:
It just calls Bash and does nothing else?  Does it not actually run any tests?

Reply:
ah no, so the command it's actually running is up above: I just want to make sure that on all 3 OSes it's using bash (otherwise it defaults to system default, which is cmd on windows and might even be zsh on macos).  regardless I only want to write the mv command once and I think windows cmd's mv won't let you just overwrite in a single command